File: app_torneig_flutter/lib/models/equip_simple_dto.dart
class EquipSimpleDTO {
  int? id;
  String? nom;
  String? curs;
  String? nomTemporada;
  int? idTemporada;
  String? imatge;
  bool? esGuanyador;

  EquipSimpleDTO({
    required this.id,
    this.nom,
    this.curs,
    this.nomTemporada,
    this.idTemporada,
    this.imatge,
    this.esGuanyador,
  });

  EquipSimpleDTO.fromJSON(Map<String, dynamic> objecteJSON) {
    id = objecteJSON['idEquip'];
    nom = objecteJSON['nom'] ?? "";
    curs = objecteJSON['curs'] ?? "";
    nomTemporada = objecteJSON['nomTemporada'] ?? "";
    idTemporada = objecteJSON['idTemporada'];
    imatge = objecteJSON['imatge'] ?? "assets/images/defecte.png";
    esGuanyador = objecteJSON['esGuanyador'];
  }
}

File: Torneig_Mort_Spring/src/main/java/org/eljust/DTO/EquipSimpleDTO.java
package org.eljust.DTO;

import lombok.Getter;
import lombok.Setter;

/**
 * Equip DTO que ens serveix per a mostrar informació més simple d'un o més
 * equips. L'utilitzarem per llistar tots els equips de la BD i els Equips
 * donada una Temporada. També per a quan volem buscar pel nom.
 */
@Getter
@Setter
public class EquipSimpleDTO {

	private Long idEquip;
	private String nom;
	private String curs;
	private String nomTemporada;
	private Long idTemporada;
	private String imatge;
	private boolean esGuanyador;
}
